src/.vuepress/components/Wasmer.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
Let's name this file `Ws.vue` to keep the table using it a bit narrower. I may then later rename `Wa` to `Wt` to avoid confusion.